chpl/chpl-api/e2e/collections/search-controller.postman_collection.json

```diff
@@ -0,0 +1,242 @@
+{
+	"info": {
+		"name": "search-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [{
+			"name": "GET /search/beta - with Compliance filters",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"exec": [
+						"pm.test(\"/search/beta end point should return 200 status and response should have Direct Review fields when Direct Reviews are available\", function () {\r",
+						"    var jsonData = pm.response.json();\r",
+						"    if ((jsonData.directReviewsAvailable) == true){\r",
+						"        pm.response.to.have.status(200);\r",
+						"        pm.expect(jsonData.results.length).not.eql(0);\r",
+						"        pm.expect(jsonData.results[17]).to.have.property('directReviewCount');\r",
```

Comment:
What does jsonData.results[17] pulls?
Looks like it is a listing but then jsonData.results[18] pulls another listing.  Why can't we have test looking for DR related properties in a listing instead of one DR properties in each different listings?

Reply:
jsonData.results array pulls different DR related properties. Since 'Compliance' filter is applied in this search test , all listings in the response should have DR properties